drivers/fs.go

```diff
@@ -121,6 +121,10 @@ func (ostore *FSSession) ListFiles(ctx context.Context, dir, delim string) (Page
 	return pi, nil
 }
 
+func (ostore *FSSession) DeleteFile(ctx context.Context, name string) error {
+	return os.Remove(name)
```

Comment:
Need to join name with prefix here as well

Reply:
Thanks, I'd forgotten that it was based on the base. I've updated the unit test too.